src/hevm/src/EVM/Transaction.hs

```diff
@@ -143,10 +202,13 @@ initTx vm = let
     touched = if creation
               then [origin]
               else [origin, toAddr]
-    accessed = fromList $ [origin, toAddr] ++ [1..9]
+    accesslist = view (EVM.tx . EVM.accessList) vm
```

Comment:
Wouldn't it be more straightforward to initialize accessAddrs and accessedStoragekeys in `makeVm` rather than here?

Reply:
done!